### DISCUSSION TITLE
MySQL import configuration broken in client in 5.4.104

### DISCUSSION THREAD

**Discussion by nickchampion** [ravendb/ravendb#16565] [General]
Hi,

Trying to set up a SQL import from MySQL and on version 5.4.104 I cannot set a connection string, see below

<img width="1949" alt="image" src="https://github.com/ravendb/ravendb/assets/3949005/f21a2286-e9e1-4ef2-b094-115e5d770c22">

This works on version 5.4.103

Are we able to create MySQL import configurations via the Node.js client while this is fixed, or any other suggestions for how we can work around this please.

Thanks

**Comment by poissoncorp** [comment]
Hi @nickchampion ,
We discovered this known issue a few days ago and are about to push the fix, which has already been finished.
I'll keep you posted on the changes, linking a pull request.

I've created an issue on our bug tracker, you can track the progress here - https://issues.hibernatingrhinos.com/issue/RavenDB-20588/MySQL-import-configuration-broken
Thanks :)
